poseidon2/types.nim

```diff
@@ -18,9 +18,17 @@ func getZero*() : F =
   setZero(z)
   return z
 
+func getOne*() : F = 
+  var y : F
+  # y.fromUint(1'u32)       # WTF, why does this not compile ???
+  y.fromHex("0x01")
+  return y
+
+# for some reason this one does not compile... ???
+# (when actually called)
```

Comment:
It is called in `testPoseidon2.nim` ?

Reply:
Yeah this was the same issue as above (it's not working at compile time)